File: NX1926_NX1930_NX1934_NX1938_NX1942_NX1946/UGOPEN/SampleNXOpenApplications/Python/MultipleDisplayParts/MDP_Off.py
#===================================================================================
# 
#       Copyright 2014 Siemens Product Lifecycle Management Software Inc. 
#                           All Rights Reserved.
# 
#   
#===================================================================================
# File Description:
#  MDP example turning MDP off
#
#===================================================================================
#

import NXOpen

def main():

    theSession = NXOpen.Session.GetSession()
    theUI = NXOpen.UI.GetUI()
    theMessageBox = theUI.NXMessageBox

    theSession.Parts.SetAllowMultipleDisplayedParts(False)

    theMDPStatus = theSession.Parts.AllowMultipleDisplayedParts
	
    theStatusString =""
	
    if theMDPStatus == NXOpen.PartCollection.MultipleDisplayedPartStatus.Disabled:
        theStatusString = "Disabled"
    elif theMDPStatus == NXOpen.PartCollection.MultipleDisplayedPartStatus.DisabledByCustomer:
        theStatusString = "DisabledByCustomer"
    else: 
        theStatusString = "Enabled"
	
    theMessageBox.Show(
        "Multiple Display Part Status", 
        NXOpen.NXMessageBox.DialogType.Information,
        "The status is {}.".format(theStatusString))

if __name__ == "__main__":
    main()

File: NX1899_NX1903_NX1907_NX1911_NX1915_NX1919/UGOPEN/SampleNXOpenApplications/.NET/MultipleDisplayParts/MDP_Off.vb
' ===================================================================
'
'             Copyright (c) 2016  Siemens PLM 
'               Unpublished - All rights reserved
'
' ===================================================================
' *******************************************************************
'  Description

'
'  
'
' *******************************************************************
Imports System
Imports NXOpen


Module NXJournal
Sub Main (ByVal args() As String) 

	Dim theSession As NXOpen.Session = NXOpen.Session.GetSession()
	Dim theUI As NXOpen.UI = NXOpen.UI.GetUI()
	Dim theMessageBox = theUI.NXMessageBox

	theSession.Parts.SetAllowMultipleDisplayedParts(false)

	Dim theMDPStatus As NXOpen.PartCollection.MultipleDisplayedPartStatus = theSession.Parts.AllowMultipleDisplayedParts

	theMessageBox.Show("Multiple Display Part Status",  NXOpen.NXMessageBox.DialogType.Information, "The status is " +theMDPStatus.toString())


End Sub
End Module
